src/test/resources/connections/list-connections-response.json

```diff
@@ -33,8 +36,7 @@
       "metadata":
       {
         "self": "http://localhost:26637/gateway/v1/connections/2",
-        "resource_name": null,
-        "sign_in_uri": "https://login.confluent-dev.io/oauth/authorize?response_type=code&code_challenge_method=S256&code_challenge=f0883e0c8257e80f15d993a26e498da93ce2e1922a04cdc54bce6a0de0127281&state=2&client_id=cUmAgrkbAZSqSiy38JE7Ya3i7FwXmyUF&redirect_uri=http://127.0.0.1:26637/gateway/v1/callback-vscode-docs&scope=email+openid+offline_access"
+        "resource_name": null
```

Comment:
Do we want to include the attribute despite being `null`?

Reply:
FYI: I've found that removing the above with my other changes didn't fail some tests, because of a regression I incorporated into my code. I'm working on a fix locally, but will probably get to it tomorrow. Thanks for pointing this out.

Reply:
I ended up keeping the original code. The test cases don't compare the `sign_in_uri` values anyway (since one component of the URL changes every time), and my new assertion logic ensures actual and expected values are either both null or both non-blank.